ddtrace/contrib/botocore/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 .. py:data:: ddtrace.config.botocore['distributed_tracing']
```

Comment:
I'm used to using environment variables like described on like 25 but how does line 19 work? Where does that line get placed? Also i'm assuming its a one or another situation, i.e. you use line 19 or an env var to enable distributed tracing.

Also is this some sort of config for the entire tracer? Is it like a toggle so that if the user wants distributed tracing then they must first enable it? Or how does that work?

Reply:
Yeah, there are two ways to modify whether `distribued_tracing` is enabled or not. Take a look here: https://ddtrace.readthedocs.io/en/stable/integrations.html?highlight=sqs#id18

The line you highlighted shows how to modify it using the tracer config, which is something you can pass when you initialize the tracer. Take a look here for an example how to pass a config to the tracer: https://ddtrace.readthedocs.io/en/stable/advanced_usage.html#usage

So, for this option, you'd pass something like:
```
{
   "botocore":{
      "distributed_tracing":false
   }
}
```

Reply:
That makes sense, thanks for the correction